GoodreadsEDA/imdb_release.py

```diff
@@ -1,85 +1,68 @@
-import pandas as pd
 import os
 import re
 import sqlite3
-
-from selenium import webdriver
-from selenium.common.exceptions import NoSuchElementException
+import traceback
 from imdb import IMDb
+from selenium import webdriver
 # my imports
-from lists import movie_titl
+from lists import movie_titles
 
 
 def create_release_table(c):
-    """Creates a table called "Release_Dates" in the reviews.db databse if it doesn't
+    """Creates a table called "Release_Dates" in the reviews.db database if it doesn't
     already exist. If "Release_Dates" does exists, it informs the user that it will
     overwrite the current database."""
 
     try:
-        c.execute(f"""CREATE TABLE Release_Dates (
-                      book_title TEXT(100) NOT NULL,
-                      movie_title TEXT(100) NOT NULL,
-                      release_date TEXT(100) NOT NULL
-                      );""")
-    except sqlite3.OperationalError as e:
-        c.execute(f"""DROP TABLE Release_Dates;""")
-        print("table being overwritten")
-        c.execute(f"""CREATE TABLE Release_Dates (
-                      book_title TEXT(100) NOT NULL,
-                      movie_title TEXT(100) NOT NULL,
-                      release_date TEXT(100) NOT NULL
-                      );""")
+        c.execute("""CREATE TABLE Release_Dates (
+                     book_title TEXT(100) NOT NULL,
+                     movie_title TEXT(100) NOT NULL,
+                     release_date TEXT(100) NOT NULL
+                     );""")
+    except sqlite3.OperationalError:
+        traceback.print_exc()
+        c.execute("""DROP TABLE Release_Dates;""")
```

Comment:
Could also do something here like `DROP TABLE Release_Dates IF EXISTS;` and drop the `try/except` block here

Reply:
Done and Done.